golem-cli/src/model/app.rs

```diff
@@ -1210,14 +1233,16 @@ mod app_builder {
                             |validation| {
                                 if invalid_source {
                                     validation.add_error(format!(
-                                        "WASM RPC dependency {} - {} references unknown component",
+                                        "{} {} - {} references unknown component",
```

Comment:
should there be validation for cycles for library deps?

Reply:
Would not hurt, I guess :)